### PR TITLE
#2319 parse @Example from @APiResponse annotations

### PR DESCRIPTION
**Pull request by pnrakesh**
Parse @Example nested inside @ApiResponse annotation

### CODE REVIEW

File: modules/swagger-jaxrs/src/main/java/io/swagger/jaxrs/Reader.java [hunk 1]
```diff
@@ -1001,9 +987,10 @@ private void processOperationDecorator(Operation operation, Method method) {
 
     private void addResponse(Operation operation, ApiResponse apiResponse) {
         Map<String, Property> responseHeaders = parseResponseHeaders(apiResponse.responseHeaders());
+        Map<String, Object> examples = parseExamples(apiResponse.examples());
 
         Response response = new Response()
-        .description(apiResponse.message()).headers(responseHeaders);
+        .description(apiResponse.message()).headers(responseHeaders).setExamples(examples);
```
Comment (frantuma) [inline modules/swagger-jaxrs/src/main/java/io/swagger/jaxrs/Reader.java]: would call `response.setExamples()` on new line, with no need of modifying setter behaviour in response POJO

File: modules/swagger-models/src/main/java/io/swagger/models/Response.java [hunk 2]
```diff
@@ -67,8 +67,9 @@ public void setSchema(Property schema) {
         return this.examples;
     }
 
-    public void setExamples(Map<String, Object> examples) {
+    public Response setExamples(Map<String, Object> examples) {
         this.examples = examples;
+        return this;
```
Comment (frantuma) [inline modules/swagger-models/src/main/java/io/swagger/models/Response.java]: we don't want to modify setter behaviour, see also comment above

File: modules/swagger-jaxrs/src/main/java/io/swagger/jaxrs/Reader.java (hunk 1 above)
Comment (pnrakesh) [reply modules/swagger-jaxrs/src/main/java/io/swagger/jaxrs/Reader.java]: Sure, I will change that.

File: modules/swagger-models/src/main/java/io/swagger/models/Response.java (hunk 2 above)
Comment (pnrakesh) [reply modules/swagger-models/src/main/java/io/swagger/models/Response.java]: Sure.

**Comment by pnrakesh** [comment]
There is already an unit test for this feature at `swagger-core\modules\swagger-core\src\test\java\io\swagger\ResponseExamplesTest.java`

**Comment by frantuma** [comment]
included and replaced by #2801

**Pull request closed without merging**